plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/impls/redshift/PostgreServerRedshift.java

```diff
@@ -83,7 +83,193 @@ public PostgreServerRedshift(PostgreDataSource dataSource) {
         "TEMP",
         "UNLOAD",
         "VACUUM",
-        "YES"
+        "YES",
+        "AES128",
+        "AES256",
+        "ALL",
+        "ALLOWOVERWRITE",
+        "AES128",
+        "AES256",
+        "ALL",
+        "ALLOWOVERWRITE",
+        "ANALYSE",
+        "ANALYZE",
+        "AND",
+        "ANY",
+        "ARRAY",
+        "AS",
+        "ASC",
+        "AUTHORIZATION",
+        "AZ64",
+        "BACKUP",
+        "BETWEEN",
+        "BINARY",
+        "BLANKSASNULL",
+        "BOTH",
+        "BYTEDICT",
+        "BZIP2",
+        "CASE",
+        "CAST",
+        "CHECK",
+        "COLLATE",
+        "COLUMN",
+        "CONNECT",
+        "CONSTRAINT",
+        "CREATE",
+        "CREDENTIALS",
+        "CROSS",
+        "CURRENT_DATE",
```

Comment:
Are you sure that this is not a function as other current_%? Because we have a separate list for functions.